TestingStaff/tstStaff.cs

```diff
@@ -7,6 +7,13 @@ namespace TestingStaff
     [TestClass]
     public class tstStaff
     {
+
+        // TEST DATA
+        string StaffPassword = "password";
```

Comment:
These are static constants, and have no accessability modifier.

* Public / private / protected
* Static
* readonly
* SNAKE_CASE_NAMES (ctrl + r twice on a symbol to refactor it)